stdlib/pervasives.md

```diff
@@ -482,6 +482,39 @@ Returns:
 |----|-----------|
 |`Number`|The modulus of its operands|
 
+### Pervasives.**(**)**
```

Comment:
This shouldn't be your fault, but this doesn't seem to be escaped correctly. Not sure what is happening.

Reply:
I've merged in main to see if the graindoc is out of date.

Reply:
I specifically rebuilt the compiler and regen'd the docs before pushing the last region commit so that probably isn't the case.

Reply:
You are correct. Markdown.bold wasn't be used in all the cases where we were bolding things 🤦

Reply:
Fixed in main since #1695 landed